SU-HSDO/suhumsci
# feat (STN-539): refine mobile styles for the Colorful carousel

# READY FOR REVIEW

## Summary
This PR incorporates refinements to the Colorful themed Carousel on mobile that compliment the recent changes to the desktop version.

* Bump the image up so that it is flush with the top of the carousel.
* Reduce the height of the color block accent.
* Reduce the width of the gray background to expose the white page background. This helps create the layered or stacked feeling that is implemented on the desktop version.

## Need Review By (Date)
9/11/2020

## Urgency
medium

## Steps to Test
1. In your CLI, run `npm run test` and confirm all our tests pass.
2. Set the theme to `Colorful`.
3. Sync your local environment with the data on staging to pull down the latest update to the [Carousel Hero Image](https://sparkbox-sandbox-stage.stanford.edu/carousel-hero-image) page.
4. Go to http://sparkbox-sandbox.suhumsci.loc/qa/carousel-hero-image
5. Reduce the browser screen width to review the mobile carousel styles.
